scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Redundant. Explain to the users what extra libraries are needed would be more helpful (packages like <code>datetime</code>  are standard, but not <code>bokeh</code>  and <code>google.cloud</code>  ).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I replaced the "Setup" heading with "Import Packages" to make things simplified. </p>